src/main/java/cn/nukkit/block/Block.java

```diff
@@ -541,8 +544,12 @@ public static Block get(int id, Integer meta) {
         return get(id, meta, null);
     }
 
-    @SuppressWarnings("unchecked")
     public static Block get(int id, Integer meta, Position pos) {
+        return get(id, meta, pos, true);
+    }
+
+    @SuppressWarnings("unchecked")
+    public static Block get(int id, Integer meta, Position pos, boolean safe) { //due to 0.15 block crash
```

Comment:
Client bug, wait for fix in 0.15.1. This is some crashes happen even in a single game.